cmd/cloud/deployment_workerqueue.go

```diff
@@ -0,0 +1,55 @@
+package cloud
+
+import (
+	"io"
+
+	"github.com/spf13/cobra"
+
+	"github.com/astronomer/astro-cli/cloud/deployment/workerqueue"
+)
+
+func newDeploymentWorkerQueueRootCmd(out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "worker-queue",
+		Aliases: []string{"wq"},
+		Short:   "Manage deployment worker queues",
+		Long:    "Manage worker queues for an Astro Deployment.",
+	}
+	cmd.AddCommand(
+		newDeploymentWorkerQueueCreateCmd(out),
+	)
+	return cmd
+}
+
+func newDeploymentWorkerQueueCreateCmd(out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "create",
+		Aliases: []string{"cr"},
+		Short:   "Create a Deployment's worker queue",
+		Long:    "Create a worker queue for an Astro Deployment",
+		Example: "", // TODO should we add some examples for default and custom worker-queue commands?
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return deploymentWorkerQueueCreate(cmd, args, out)
+		},
+	}
+	cmd.Flags().StringVarP(&deploymentID, "deployment-id", "d", "", "The deployment where the worker queue should be created.")
+	cmd.Flags().StringVarP(&deploymentName, "deployment-name", "", "", "Name of the deployment where the worker queue should be created.")
+	cmd.Flags().StringVarP(&wQueueName, "name", "n", "", "The name of the worker queue. Queue names must not exceed 63 characters and contain only lowercase alphanumeric characters or '-' and start with an alphabetical character.")
+	cmd.Flags().IntVarP(&wQueueMinWorkerCount, "min-count", "", 0, "The min worker count of the worker queue. Possible values are between 1 and 10.")
+	cmd.Flags().IntVarP(&wQueueMaxWorkerCount, "max-count", "", 0, "The max worker count of the worker queue. Possible values are between 11 and 20.")
+	cmd.Flags().IntVarP(&wQueueConcurrency, "concurrency", "", 0, "The concurrency(number of slots) of the worker queue. Possible values are between 21 and 30.")
```

Comment:
In the UI possible values are between 1 and 64

Reply:
@sunkickr I have removed the `"Possible values...` from the min and max help text so we do not have to hard code any values here. The CLI gets these values from astrohub and eforces them. If you are out of the allowed range an error is returned showing you what the allwoed range is. 

Same as min and max values above. Let me know if that works better.

Reply:
I think that works better thanks